script.js

```diff
@@ -1,4 +1,183 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+// Version 2 Comfortable - (Mon - Thu: 2h - 3h)
+/*
+- Player input number of dice, number of player and then select game mode
+- Press start button to display the player and the player's optimal combination
+- The highest / Lowest combination wins
+*/
+const diceInput = document.querySelector("#dice-number");
+const playerInput = document.querySelector("#player-number");
+const gameModeInput = document.querySelector("#game-mode");
+const submitBtn = document.querySelector("#submit-button");
+const outputDiv = document.querySelector("#output-div");
+const resetBtn = document.querySelector("#reset-button");
+resetBtn.style.display = "none";
```

Comment:
good use of dom manipulation, would recommend exploring this further in blackjack before bootcamp